doc/poo.md

```diff
@@ -350,6 +350,17 @@ In that list:
     since those in the current prototype will already be implicitly included
     in the list of symbols to bind.
 
+    As an example, we have a `slot-name` x, which is bound in the macro `make-o-with`:
+    ```lisp
+    (def o {(:: @ [] y)
+      x: y}) ;; what is y?
```

Comment:
Here, `y` is a lazy access to another slot named `y`. If you try to compute `(.@ o x)` it will fail saying that the slot is unbound. If you compose `o` with another prototype that defines `y` and doesn't override `x`, then `x` will be bound to the same value as `y`.